dev-libs/aws-c-common/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<longdescription>
```

Comment:
Please follow the metadata skel and put longdescription after maintainers. I'm asking to make this change because there are a lot of scripts that crawl the metadata info, and while I doubt this will break anything, let's make sure we are consistent with other in-tree packages.

Reply:
Check